docs/payments.md

```diff
@@ -0,0 +1,25 @@
+# Payments
+
+The Venepaikka payments rely mostly on the [Respa](https://github.com/City-of-Helsinki/respa/blob/develop/docs/payments.md) implementation, however, changes were made to adapt it to the requirements of the project.
+
+## Configuring Payments
+
+There are a couple of required configuration keys that need to be set in order to enable the payments and the third party provider currently in use:
+
+- `VENE_PAYMENTS_PROVIDER_CLASS`: Dotted path to the active provider class as a string. The default value is `payments.providers.BamboraPayformProvider`.
+
+`./manage.py expire_too_old_unpaid_orders` runs the order/reservation cleanup for current orders. You'll probably want to run it periodically at least in production. [Cron](https://en.wikipedia.org/wiki/Cron) is one candidate for doing that.
```

Comment:
One thing to consider: Cron is not ideal solution with Docker. It would be great if we could aim for solution that would play well with KuVa infra (Docker).

Reply:
We can set a [Kubernetes CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) that runs the command.